index.html

```diff
@@ -38,6 +38,8 @@ <h1>Getting Started with JavaFX 11</h1>
                         <a class="list-group-item" href="#install-javafx">Run HelloWorld using JavaFX 11</a>
                         <a class="list-group-item" href="#maven">Run HelloWorld via Maven</a>
                         <a class="list-group-item" href="#gradle">Run HelloWorld via Gradle</a>
+                        <a class="list-group-item" href="#fat-jar">Creating a fat Jar</a>
+                        <a class="list-group-item" href="#modular">Creating modular projects</a>
```

Comment:
I would switch modular to come before fat jar.

Reply:
Makes sense, we prefer and recommend modular, though mainly people look for solutions for non-modular apps. Let's see what others say about this.

Reply:
+1 on switching

Reply:
I don't think most of the people are looking for non-modular apps. It's probably just that those people run into issues more easily.